modals-notifications/es5/src/js/notification.js

```diff
@@ -0,0 +1,55 @@
+function Notification(config, duration) {
+	Parent.prototype.constructor.apply(this, arguments);
+
+	this.parent = document.querySelector('.notificationsJS');
```

Comment:
The same here. You either use querySelectorAll and work off all the matches, or pass element itself as an argument